bn-apps/cordapp-updates-distribution/corda-updates-transport/src/main/kotlin/com/r3/businessnetworks/cordaupdates/transport/flows/AbstractRepositoryHosterResponder.kt

```diff
@@ -17,16 +17,21 @@ import net.corda.core.serialization.CordaSerializable
 abstract class AbstractRepositoryHosterResponder<T>(val session : FlowSession) : FlowLogic<T>() {
     @Suspendable
     override fun call() : T {
-        val configuration = serviceHub.cordaService(RepositoryHosterConfigurationService::class.java)
-        val sessionFilter = configuration.getSessionFilter()
-        if (sessionFilter != null && !sessionFilter.isSessionAllowed(session, this)) {
+        if (!isSessionAllowed(session)) {
             throw FlowException("Counterparty ${session.counterparty} is not allowed to access repository")
         }
         return postPermissionCheck()
     }
 
     @Suspendable
     protected abstract fun postPermissionCheck() : T
+
+    /**
+     * Override this method to implement session filtering
+     * See: https://docs.corda.net/head/flow-overriding.html
+     */
+    @Suspendable
+    protected open fun isSessionAllowed(session : FlowSession) = true
```

Comment:
Should this be `abstract fun isSessionAllowed` without a body to force implementations to explicitly state they want to allow all sessions?

Reply:
The original idea was to allow people to use this service out-of-the-box, without a need to provide any custom overrides. Maybe they are running their private network in intranet and don't care about any custom verifications, who knows.